src/s2client-fsharp/Railway.fs

```diff
@@ -0,0 +1,101 @@
+﻿namespace Rail
+
+module Result =
```

Comment:
Maybe we should change the namespace and mark that module `internal`?

Reply:
Agreed.